app/controllers/api/students_controller.rb

```diff
@@ -10,20 +10,20 @@ def create
   end
 
   def index
-    students = Student.includes(:school).page params[:page]
+    students = Student.includes(:group, :school).page params[:page]
```

Comment:
`.includes` tells the query to also grab related `group` and `school` when grabbing a `student` record. This saves us queries when the serializers try to follow their respective `has_one`s.